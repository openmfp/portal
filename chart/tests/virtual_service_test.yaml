suite: virtual service
templates:
  - virtual-service.yaml
values:
  - ../test-values.yaml
tests:
  - it: virtual service match the snapshot
    asserts:
      - matchSnapshot: {}
