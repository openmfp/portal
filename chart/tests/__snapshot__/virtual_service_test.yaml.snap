virtual service match the snapshot:
  1: |
    apiVersion: networking.istio.io/v1alpha3
    kind: VirtualService
    metadata:
      name: RELEASE-NAME
      namespace: NAMESPACE
    spec:
      gateways:
        - NAMESPACE/gateway
      hosts:
        - portal.example.com
        - portal.portal.example.com
        - portal2.example.com
        - portal.portal2.example.com
      http:
        - corsPolicy:
            allowHeaders:
              - Authorization
              - Content-Type
              - '*'
            allowMethods:
              - GET
              - POST
              - PUT
              - DELETE
          route:
            - destination:
                host: RELEASE-NAME.NAMESPACE.svc.cluster.local
                port:
                  number: 8080
